src/components/sbb-toast/sbb-toast.tsx

```diff
@@ -150,17 +158,40 @@ export class SbbToast implements ComponentInterface {
     this.willClose.emit();
   }
 
-  @Method() public async getState(): Promise<SbbOverlayState> {
-    return this._state;
+  @Listen('click') public async onClick(ev: Event): Promise<void> {
+    if ((ev.target as HTMLElement).hasAttribute('sbb-toast-close')) {
```

Comment:
I think it would be a little bit more safe to do the same approach as in tooltip by reading the composedPath. What do you think?

```
event
      .composedPath()
      .find(
        (e) => e instanceof window.HTMLElement && e.hasAttribute('sbb-toast-close')
      ) as HTMLElement;
```

Reply:
It would include also shadow dom cases. e.g. in sbb-tooltip we applied "sbb-tooltip-close" also to the internal close button (could be done here as well). the parents should better not be included as you said. Lukas and I had the idea of creating a helper function doing this. something like `composedPathHasAttribute(event, attributeName, endElement)`. This function would filter out all elements after/above endElement and then check for an attribute. And as you mentioned it would also include setting sbb-tooltip-close on in between elements, like on a container div containing several actions.

Reply:
Thanks for the explanation :D
I'll align the implementation to the tooltip

Reply:
I think we had a little misunderstanding here, can you create a helper function for both the tooltip and toast as following: `composedPathHasAttribute(event, attributeName, endElement)` (or similar). The function should exclude all parents (starting from endElement which would be sbb-tooltip, or sbb-toast). This fixes a rare case issue where parent elements are searched too and helps avoiding code duplication. Thanks!

Reply:
My bad :( 
I thought that the helper method was a future refactoring. I'll fix this asap